WordPress/src/main/java/org/wordpress/android/ui/compose/components/menu/dropdown/MenuColors.kt

```diff
@@ -0,0 +1,30 @@
+package org.wordpress.android.ui.compose.components.menu.dropdown
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import org.wordpress.android.ui.compose.theme.AppColor
+
+@Composable
+fun isLightTheme(): Boolean = MaterialTheme.colors.isLight
+
+@Composable
+fun itemContentColor(): Color = if (isLightTheme()) {
+    AppColor.Black
+} else {
+    AppColor.White
+}
```

Comment:
I agree we should rely on `MaterialTheme.colors` whenever possible, but the theme never worked as expected while using Cascade 😓 I've tried for some time using `MaterialTheme.colors.<color>` and it would work on one theme (e.g. light) but not in the other.

I'll quickly try again, but I really don't have new ideas at this point.

Reply:
Given the latest code is still using this I will guess it was not possible to use the colors directly from MaterialTheme, is that right? If that's the case feel free to close this comment as resolved.